assets/webconfig/js/content_leds.js

```diff
@@ -472,11 +472,8 @@ $(document).ready(function() {
 	
 	// ------------------------------------------------------------------
 	$("#leds_custom_save").off().on("click", function() {
-		function createLedConfig(){
-			var string = '{"leds" :'+$("#ledconfig").val()+'}';
-		}
 		if (validateText())
-			requestWriteConfig(JSON.parse(createLedConfig()));
+			requestWriteConfig(JSON.parse('{"leds" :'+$("#ledconfig").val()+'}'));
```

Comment:
uhm, of course. That was my intention. For a unknown reason i stopped in between :)